input/fsh/OrganizationSEVendorLite.fsh

```diff
@@ -60,7 +60,7 @@ Description: """This is a light weight Swedish Organization profile.
     and identifier[hsaid].value
     and identifier[organizationIdentifier]
     and identifier[organizationIdentifier].system
-    and identifier[organizationIdentifier].value
+    and identifier[organizationIdentifier].value obeys organisationsnummer-invariant
```

Comment:
This won't work. Please keep the line 63 as it is for define MS elemnts. suggests to define the rule in a separate line after define MS elemnts.
`* identifier[organizationIdentifier].value obeys organisationsnummer-invariant`